app/Http/Controllers/CalendarController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Conference;
+use Calendar;
+
+class CalendarController extends BaseController
+{
+    public function index()
+    {
+        $conferences = Conference::all()->map(function ($conference) {
```

Comment:
Yeah, I generally try to avoid private controller methods like the plague, but it doesn't feel quite right on the Conference class either. I think private methods are good for now. Maybe as the calendar functionality grows later on, the proper pattern will become more clear.